src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,12 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos, users }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo
+        todo={todo}
+        key={todo.id}
+      />
+    ))}
+  </section>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings